frontend/src/components/common/NamespacesAutocomplete.tsx

```diff
@@ -15,10 +15,18 @@ import Namespace from '../../lib/k8s/namespace';
 import { setNamespaceFilter } from '../../redux/filterSlice';
 import { useTypedSelector } from '../../redux/reducers/reducers';
 
+const inputRef = React.createRef<HTMLInputElement>();
+
+export function openNamespaceAutocomplete() {
+  setTimeout(() => {
+    inputRef.current?.focus?.();
+  }, 0);
+}
+
 export interface PureNamespacesAutocompleteProps {
   namespaceNames: string[];
   onChange: (event: React.ChangeEvent<{}>, newValue: string[]) => void;
-  filter: { namespaces: Set<string>; search: string };
+  filter: { namespaces: Set<string> };
```

Comment:
same as above

Reply:
It's not clear to me if it matters that there is an API change here.

Reply:
@illume I discussed with @sniok that given that the table search was actually getting more powerful than the redux based search we have + avoided the problem we have now that we can only use one instance of the search per view, otherwise it will reflect in all tables.

Reply:
Ok. 

Does the API change matter here? I don't know if anyone is using this from plugins or not.

Reply:
typecheck won't fail because {namespace, search} will still be assignable to {namespace}. and search wasn't even used in this component anyway, this is just a namespace selector.